File: Models/Data/Atributes/Atribute.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ProductsReviewsAngular.Models
{
    public class Atribute
    {
        public int idAtribute { get; set; }
        [ConcurrencyCheck]
        [MaxLength(50)]
        public string name { get; set; }
        public bool isEnable { get; set; }
        public bool isCanBeMany { get; set; }

        public int idAtrbutesGroup { get; set; }
        public virtual AtributesGroup AtributesGroup { get; set; }

        public virtual IEnumerable<AtributeValue> AtributeValues { get; set; }
    }
}

File: ClientApp/src/app/models/Atribute.ts
import {AtributeValue} from "./AtributeValue";
export class Atribute{    
    public idAtribute?: number;
    public name?: string;
    public AtributeValues?: AtributeValue[];
    public isCanBeMany?: boolean;

    constructor(params: Atribute = {} as Atribute){
        let{
            idAtribute = undefined,
            name = undefined,
            AtributeValues = [],
            isCanBeMany = false
        } = params;
        this.idAtribute = idAtribute;
        this.name = name;
        this.AtributeValues = AtributeValues;
        this.isCanBeMany = isCanBeMany;
    }
}